src/render/tutorials/configurable-api-gateway/2_0.html.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: Configurable API gateway
-author: marcinkp
+author: mateuszgrab-cognifide
```

Comment:
I am not sure we can change the author when we do some improvements. @mateuszgrab-cognifide: I hope you are not disappointed.

Reply:
Sure, understood, I've recovered original author.